cmd/main.go

```diff
@@ -37,6 +38,8 @@ func main() {
 	port := os.Getenv("API_SERVER_PORT")
 	serv := new(server.Server)
 
+	auth.GenerateSecret()
```

Comment:
It was possible to move this in `init` function of the package. Or just in variable initialization.

Reply:
fixed